.dockerignore

```diff
@@ -0,0 +1,8 @@
+kube
```

Comment:
Yeah sure. (Maybe "deploy" to be shorter?)
I still think you should have a soforms-devops repo or something. I'd think the deployment would be identical between most of the forms.